VirtoCommerce.Storefront/Controllers/AccountController.cs

```diff
@@ -737,8 +738,8 @@ public async Task<ActionResult> ResetPassword(string token, string userId)
         public async Task<ActionResult> ResetPassword(ResetPassword formModel)
         {
             TryValidateModel(formModel);
-            //Need reassign the passed form to the current context to allow for user post it again with initial data such as Token and Email
-            WorkContext.Form = Form.FromObject(formModel);
+            // Need reassign the passed form to the current context to allow for user post it again with initial data such as Token and Email
+            // WorkContext.Form = Form.FromObject(formModel);
```

Comment:
This line is required for normal form validation on the theme. You can't comment on it.